remo/voting/helpers.py

```diff
@@ -47,3 +48,11 @@ def user_has_voted(user, poll):
     if Vote.objects.filter(poll=poll, user=user).exists():
         return True
     return False
+
+
+@register.function
+def get_nominee(full_name):
+    first_name, last_name = full_name.split()
```

Comment:
I believe there is no need for this since all the names are directly from our database, in which they are already normalized.